UnitTests/TransportTests.cpp

```diff
@@ -18,6 +19,8 @@
 
 BOOST_AUTO_TEST_SUITE(TransportTests)
 
+using Traits = IPC::UnitTest::Mocks::NullTimeoutTraits;

```

Comment:
I use `Traits` alias in tests so that it will be easier to change its type when needed